releasetool/commands/tag/ruby.py

```diff
@@ -79,10 +79,12 @@ def determine_package_name_and_version(ctx: TagContext) -> None:
 
 def get_release_notes(ctx: TagContext) -> None:
     click.secho("> Grabbing the release notes.", fg="cyan")
+    if ctx.upstream_repo == ctx.package_name:
```

Comment:
I'm pretty sure `ctx.upstream_repo` includes the repo owner. e.g., `googleapis/gax-ruby`. 

Maybe `if ctx.package_name in ctx.upstream_repo:`?

Reply:
Thanks, updated